spec/index.bs

```diff
@@ -2210,20 +2180,125 @@ enum GPUTextureAspect {
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is a multiple of `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"3d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"3d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+                                </dl>
+                        </div>
+
+                        Then:
+                            1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+                            1. Create a new [=invalid=] {{GPUTextureView}} and return the result.
+
+                    1. Let |view| be a new {{GPUTextureView}} object.
+                    1. Set |view|.{{GPUTextureView/[[texture]]}} to |this|.
+                    1. Set |view|.{{GPUTextureView/[[descriptor]]}} to |descriptor|.
+                    1. Return |view|.
+                </div>
+        </div>
 </dl>
 
+Issue: Define what it means to be a <dfn dfn>compatible texture format</dfn> with another texture format.
+
+<div algorithm>
+    When <dfn abstract-op>resolving GPUTextureViewDescriptor defaults</dfn> for {{GPUTextureViewDescriptor}}
+    |descriptor| run the following steps:
+
+    1. Let |resolved| be a copy of |descriptor|.
+    1. If |resolved|.{{GPUTextureViewDescriptor/format}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/format}} to |texture|.{{GPUTexture/[[format]]}}.
+    1. If |resolved|.{{GPUTextureViewDescriptor/mipLevelCount}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/mipLevelCount}} to |texture|.{{GPUTexture/[[mipLevelCount]]}}
+        &minus; {{GPUTextureViewDescriptor/baseMipLevel}}.
+    1. If |resolved|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/arrayLayerCount}} to
+        |texture|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=] &minus;
+        {{GPUTextureViewDescriptor/baseArrayLayer}}.
+    1. If |resolved|.{{GPUTextureViewDescriptor/dimension}} is `undefined`:
+        - If |texture|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"1d"}},
+            set |resolved|.{{GPUTextureViewDescriptor/dimension}} to {{GPUTextureViewDimension/"1d"}}.
+        - If |texture|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}:
+            - If |texture|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=] is greater than `1`
+                and |resolved|.{{GPUTextureViewDescriptor/arrayLayerCount}} is greater than `1`,
```

Comment:
Found another thing while reviewing a CTS change.

The meaning here has changed from the original (before this PR), though that was outdated because it special-cased 0. I think this should be refactored to say:

- If texture.dimension is 2d:
    - If texture.textureSize.depth is 1 or (the original, pre-defaulting) arrayLayerCount is defined, view dimension is 2d.
    - else view dimension is 2d-array.

Pretty sure that's the original intent. Though it's kind of weird? Maybe it should have been:

- If texture.dimension is 2d:
    - If texture.textureSize.depth is 1 or (the original, pre-defaulting) arrayLayerCount is 1, view dimension is 2d.
    - else view dimension is 2d-array.